frontend/app/components/transportation/tdf/modal-splits/census-tracts-map.js

```diff
@@ -80,6 +86,20 @@ export default class TransportationCensusTractsMapComponent extends Component {
   mapLoaded(map) {
     map.on('data', onMapStyleLoaded);
 
+    map.addControl(new MapboxAccessibility({
```

Comment:
Maybe this should be componentized? Just so we don't add more stuff to the map...